jelly/src/main/java/org/kohsuke/stapler/jelly/DefaultScriptInvoker.java

```diff
@@ -142,16 +144,18 @@ public void close() throws IOException {
     }
 
     protected void exportVariables(
-            StaplerRequest req, StaplerResponse rsp, Script script, Object it, JellyContext context) {
+            StaplerRequest2 req, StaplerResponse2 rsp, Script script, Object it, JellyContext context) {
         Enumeration en = req.getAttributeNames();
         // expose request attributes, just like JSP
         while (en.hasMoreElements()) {
             String name = (String) en.nextElement();
             context.setVariable(name, req.getAttribute(name));
         }
 
-        context.setVariable("request", req);
-        context.setVariable("response", rsp);
+        context.setVariable("request", StaplerRequest.fromStaplerRequest2(req));
+        context.setVariable("response", StaplerResponse.fromStaplerResponse2(rsp));
+        context.setVariable("request2", req);
+        context.setVariable("response2", rsp);
```

Comment:
I had briefly considered changing `request` and `response` into Jakarta versions, but this broke plugins with Jelly views that called plugin methods whose parameters were requests or responses.